to_do_list_part_two.py

```diff
@@ -27,22 +27,23 @@ def view_tasks(task_list):
         print("The to-do list is empty. Consider adding a new task!")
     else:
         print("To-Do List:")
-        priority_order = {'high': 1, 'medium': 2, 'low': 3}
-        sorted_tasks = sorted(task_list, key=lambda x: (priority_order[x['priority']], x['deadline']))
+        sorted_tasks = sort_tasks_by_priority_and_deadline(task_list)
         for idx, task in enumerate(sorted_tasks, start=1):
-            print(f"{idx}. {task['task']} - {task['priority']} - {task['deadline'].strftime('%Y-%m-%d')}")
+            print(f"{idx}. {task['task']} - {task['priority']} - {task['deadline']}")
```

Comment:
https://github.com/kiarash2077/To-Do-List-Application/pull/21#discussion_r1803850313

This change is not reflected in this pull request.

```suggestion
            print(f"{idx}. {task['name']} - {task['priority']} - {task['deadline']}")
```

Reply:
you mean it is not working?